pipeline/terraform/stack/service_work_relation_embedder.tf

```diff
@@ -0,0 +1,72 @@
+module "relation_embedder_input_queue" {
+  source = "git::github.com/wellcomecollection/terraform-aws-sqs//queue?ref=v1.1.2"
+
+  queue_name                 = "${local.namespace_hyphen}_relation_embedder_input"
+  topic_arns                 = [module.merger_works_topic.arn]
+  visibility_timeout_seconds = 120
+
+  aws_region      = var.aws_region
+  alarm_topic_arn = var.dlq_alarm_arn
+}
+
+module "relation_embedder" {
+  source          = "../modules/service"
+  service_name    = "${local.namespace_hyphen}_relation_embedder"
+  container_image = local.relation_embedder_image
+
+  security_group_ids = [
+    aws_security_group.service_egress.id,
+    aws_security_group.interservice.id,
+  ]
+
+  cluster_name = aws_ecs_cluster.cluster.name
+  cluster_arn  = aws_ecs_cluster.cluster.id
+
+  env_vars = {
+    metrics_namespace    = "${local.namespace_hyphen}_relation_embedder"
+    messages_bucket_name = aws_s3_bucket.messages.id
```

Comment:
Not sure what this bucket is for, maybe copy / paste from an app that uses `big_messaging`? :thinking: